product_docs/docs/pgd/5.6/nodes/subscriber_only/overview.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: An overview of Subscriber-only nodes
+navTitle: Overview
+---
+
+## Overview
+
+While many use cases rely on accessing a database node which can handle queries and updates, there are also use cases which only require access to a node that can handle read-only database queries.
+Read scaling like this, by moving the read-only traffic away from active database nodes in the cluster, can improve the performance of the core cluster, whilst making database access more widely available.
+
+The basic idea of subscriber-only nodes is to provide a read-only node that can be used to offload read-only queries from the main cluster. 
+How it connects to the other nodes in the cluster can be different from how other nodes connect to each other.
+The default topology of a PGD cluster is what we call a full mesh topology, where every node connects to every other node.
+This is the most robust and fault-tolerant way to connect nodes, but it can be inefficient for some use cases.
+
+Each node in a subscriber-only group will, by default, receive replication updates from all other nodes in the cluster.
+
```

Comment:
What is "this" on line 18th referring to?

Are these 18,19,20 lines explaining the leader based replication for SO group?
Then it again mentions similar things on line 22 but with group leader and efficient model, so a bit confusing for me.

Reply:
Fixed